tests/unit/linalg/test_constrainedsolver.cpp

```diff
@@ -0,0 +1,813 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
```

Comment:
Here and in the other new files

```suggestion
// Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
```